### ISSUE TITLE
Propagate `.gitattributes` to Nextstrain pathogen repos

### ISSUE THREAD

**Issue by joverlee521** [nextstrain/pathogen-repo-guide#46]
We had a question Nextstrain office hours where a user was running into an error with the curate rule in the ingest workflow:

```
/usr/bin/env: ‘python3\r’: No such file or directory
```

@genehack flagged that this is likely an issue with line endings in [Slack](https://bedfordlab.slack.com/archives/C7SDVPBLZ/p1718906921283479?thread_ts=1718901471.304359&cid=C7SDVPBLZ), which reminded me that we added `.gitattributes` in https://github.com/nextstrain/pathogen-repo-guide/pull/37 for this exact reason! 

We should propagate the `.gitattributes` file to all the Nextstrain pathogen repos so that Window's users are less likely to run into this error.

### TODOs

Pathogen repos from https://github.com/search?q=topic%3Apathogen+topic%3Anextstrain&type=repositories

- [x] avian-flu https://github.com/nextstrain/avian-flu/pull/76
- [x] dengue https://github.com/nextstrain/dengue/pull/81
- [x] https://github.com/nextstrain/ebola/pull/9
- [x] forecasts-ncov https://github.com/nextstrain/forecasts-ncov/pull/105
- [x] https://github.com/nextstrain/hbv/pull/16
- [x] https://github.com/nextstrain/measles/issues/29 https://github.com/nextstrain/measles/pull/51
- [x] https://github.com/nextstrain/mers/pull/1
- [x] https://github.com/nextstrain/mpox/pull/260
- [x] https://github.com/nextstrain/mumps/pull/9
- [x] ncov https://github.com/nextstrain/ncov/pull/1125
- [x] ncov-ingest https://github.com/nextstrain/ncov-ingest/pull/464
- [x] rsv https://github.com/nextstrain/rsv/pull/71
- [x] seasonal-flu -> already [exists](https://github.com/nextstrain/seasonal-flu/blob/00cfae195cc8512f31f15c662ca311e0cb63de7d/.gitattributes)
- [x] https://github.com/nextstrain/tb/pull/12
- [x] zika https://github.com/nextstrain/zika/pull/71
- [x] zika-tutorial https://github.com/nextstrain/zika-tutorial/pull/15

Other pathogen repos:

- [x] https://github.com/nextstrain/seasonal-cov/commit/ec466b554f677588500ae87e5f36072573a0e97c
- [x] https://github.com/nextstrain/yellow-fever/commit/8b67f61d46997376966925d8400fb315af5b5862
- [x] https://github.com/nextstrain/rabies/commit/ad0b045811bdd327a4caadf1f2ce9b94430dc4cb

Following [.gitattribute docs](https://git-scm.com/docs/gitattributes#_end_of_line_conversion) to normalize line endings of existing files in repos.

**Comment by genehack** [comment]
Done for `seasonal-cov` and `yellow-fever` — note that `seasonal-cov` had `.gitattributes` in `.gitignore` so that needed to be adjusted as well; not sure where that came from originally (and didn't try to track it down either...)

**Comment by joverlee521** [comment]
@genehack raised a question in the Nextstrain dev chat of whether forcing LF line endings could cause unexpected behavior for Windows users _outside_ of Docker. @victorlin is exploring behaviors with `.gitattributes` on a Windows laptop.

**Comment by victorlin** [comment]
I just tried some different scenarios detailed below. Let me know if you can think of other scenarios to test.

TLDR: text files are handled properly with `* text=auto eol=lf` or `* text eol=lf`. We settled on `* text=auto eol=lf` in https://github.com/nextstrain/pathogen-repo-guide/pull/37#discussion_r1543501665

**What does a Windows text editor do?**

I tested two common editors: Notepad (Windows version 21H1) and VS Code (1.90.2).

- Both editors detected CRLF or LF on existing files and kept consistent when making changes.
- Both editors saved new files as CRLF. This is where having the right `.gitattributes` is helpful.

**What does `.gitattributes` do?**

From [git docs](https://git-scm.com/docs/gitattributes#_checking_out_and_checking_in):

> These attributes affect how the contents stored in the repository are copied to the working tree files when commands such as **`git switch`**, **`git checkout`** and **`git merge`** run. They also affect how Git stores the contents you prepare in the working tree in the repository upon **`git add`** and **`git commit`**.

This seems to be accurate. Steps done to verify (all commands run on Windows PowerShell):

- Cloning zika (no `.gitattributes`) resulted in CRLF on all text files - no good.
- Cloning mpox (which currently has `* text eol=lf`) resulted in LF on all text files. Good!
- Cloning mpox from an old branch without `.gitattributes` resulted in CRLF on all text files. Switching branches to latest `master` resulted in LF on all text files. Good!
- Cloning pathogen-repo-guide (which currently has `* text=auto eol=lf`) resulted in LF on all text files. Good!
- New text files with `* text eol=lf` or `* text=auto eol=lf` get stored in git as LF upon `git add`. They're kept as CRLF locally but a `git switch` to a commit without the file then back to a commit that adds the file results in conversion to LF. Good!

**Comment by joverlee521** [comment]
Thanks for pushing this through to the end @genehack 🌟
